docs/mine-hnt/validators/penalties.mdx

```diff
@@ -60,16 +61,17 @@ Now that there's a high-level description of how penalties affect the election p
 
 To ensure that consensus groups members are continually churned, and that no individual Validator participates for too long, penalties are added. **This is not a performance based penalty and is added to _all_ Validators that participate in a consensus group.**
 
-* A penalty of `1.0` is added at the end of each consensus group for each Validator that participated in that group.  If the Validator is not ejected from the consensus group, it will receive another `1.0` penalty at the end of the 2nd consensus group, now having a score of `2.0`.  
-* This penalty of `1.0` is added even if the Validator is ejected from the consensus group as it is a penalty added for participating in the ending consensus group. This penalty is called the `tenure_penalty`.
-
+- A penalty of `0.5` is added at the beginning of each consensus group for each Validator that in participating in the group. If the Validator is not ejected from the consensus group, it will receive another `0.5` penalty at the beginning of the 2nd consensus group, now having a score of `1.0`.
```

Comment:
I agree with the first two sentences.  The third:

At the start of each epoch, any performance penalties that have been accumulated during the prior epoch will be applied to each member of the outgoing consensus group.

Since these other penalties haven't been defined yet on this page, I think this will just create confusion.  Maybe it's clearer to add a bullet on all penalties on when they are applied.  I'll make that change.